docs/users-guide/interactive-dashboards.md

```diff
@@ -0,0 +1,104 @@
+
+![Pass value to dashboard](images/interactive-dashboards/pass-value.png)
+
+Once you select a column, the sidebar will display the column used to pass the value, as well as the target filter at the destination that Metabase will pass the value to.
+
+![Pass category to filter](images/interactive-dashboards/pass-category-to-filter.png)
+
+In the example above, when a user clicks on the **Orders by product category** card, Metabase will pass the clicked `Category` to the destination dashboard ("Interactive Dashboard"), which will then filter its cards by that `Category`.
+
+You can also send the currently selected value of a dashboard filter on the current dashboard to the destination. And if you're using [Metabase Enterprise Edition](https://www.metabase.com/enterprise/scale/index.html), you can pass a user attribute provided by SSO to the destination, too. Those user attributes will show up as options when you click on one of the destination's filters ( provided the values are compatible with that filter).
+
+When displaying questions as tables, you can select different click behaviors for different columns in the table. You can also modify the contents of the cells in a given column, replacing the value with custom text. For example, if you had a column that listed categories, you could change the text in the cell to read: "Click for details about {{Category}}".
+
+You can also use values to construct URLs to external resources.
+
+![Enter a URL](images/interactive-dashboards/enter-a-url.png)
+
+From the **Click behavior** sidebar, select **Go to a custom destination** and link to **URL**. The **Enter a URL to link to** modal will pop up, allowing you to specify a URL, as well as specify the value of a column or dashboard filter. Click on the dropdown menu **Values you can reference** to see your options.
+
+### Use a chart to filter a dashboard
+
+If your dashboard contains at least one filter, you can set things up so that clicking on a chart in the dashboard will update a filter.
+
+When a user clicks on, say, a bar in a bar chart, you could send the value of the clicked bar to the filter, and update cards across the dashboard. We call this functionality **cross-filtering**. You can use this cross-filtering to make a chart behave as kind of "navigation question" that filters data across other cards.
+
+For example, clicking on the `Widget` bar will update the current dashboard's **category** filter to filter for `Widget`:
+
+![Cross-filtering](images/interactive-dashboards/cross-filter.png)
+
+To set up cross-filtering, choose a dashboard filter that you'd like to update on click, and a question to use to update that filter. You can think of this question as your "navigation question". Instead of wiring this navigation question up to the filter, you'll [wire up every other question on the dashboard to the filter](08-dashboard-filters.md). 
```

Comment:
```suggestion
To set up cross-filtering, choose a dashboard filter that you'd like to update on click, and a question to use to update that filter. You can think of this question as your "navigation question." Instead of wiring this navigation question up to the filter, you'll [wire up every other question on the dashboard to the filter](08-dashboard-filters.md). 
```